app/decorators/category_decorator.rb

```diff
@@ -20,10 +20,19 @@ def canonical_url(posts)
   end
 
   def page_title(posts)
+    if site.category_title_format.present?
+      begin
+        title = site.category_title_format % {category_name: name}
+      rescue KeyError
```

Comment:
> This code improves robustness.

I don't think so.
This `rescue` will hide some bugs against our expectation.

If there are some bugs, it should be raised to notify our bug tracking system.